### PR TITLE
[llm]Merge benchmark_utils.py and gpu_benchmark_utils.py

### PR DESCRIPTION
**Pull request by qiuxin2012**
## Description

Merge benchmark_utils.py and gpu_benchmark_utils.py

### 1. Why the change?

Too many duplicated code.

### 2. User API changes

None

### 3. Summary of the change 

Merge benchmark_utils.py and gpu_benchmark_utils.py.

### 4. How to test?
- [ ] N/A
- [ ] Unit test
- [ ] Application test
- [ ] Document test
- [ ] ...

### CODE REVIEW

File: python/llm/dev/benchmark/benchmark_util.py [hunk 1]
```diff
@@ -510,9 +510,12 @@ class BenchmarkWrapper:
     learn more about decoding strategies refer to the [text generation strategies guide](../generation_strategies).
     """
     
-    def __init__(self, model, do_print=True):
+    def __init__(self, model, do_print=False):
```
Comment (rnwang04) [inline python/llm/dev/benchmark/benchmark_util.py:513]: shall we leave the default value as true?

File: python/llm/dev/benchmark/benchmark_util.py [hunk 2]
```diff
@@ -510,9 +510,12 @@ class BenchmarkWrapper:
...
         self.model = model
         self.do_print = do_print
+        self.encoder_time = 0.0
+        self.first_cost = 0.0
+        self.rest_cost_mean = 0.0
```
Comment (rnwang04) [inline python/llm/dev/benchmark/benchmark_util.py:518]: it seems that these three variables may be unnecessary ?

**Review by rnwang04 — COMMENTED** [review]
others LGTM

File: python/llm/dev/benchmark/benchmark_util.py (hunk 2 above)
Comment (qiuxin2012) [reply python/llm/dev/benchmark/benchmark_util.py:518]: To resolve pycharm's warming: `instance attribute defined outside __init__`

File: python/llm/dev/benchmark/benchmark_util.py (hunk 1 above)
Comment (qiuxin2012) [reply python/llm/dev/benchmark/benchmark_util.py:513]: > shall we leave the default value as true?
From the disscusion in https://github.com/intel-analytics/BigDL/pull/8871, we want to hide the print by default.

**Review by rnwang04 — APPROVED** [review]
LGTM

**Pull request merged**